### DISCUSSION TITLE
Web Analytics: Which User Agents get filtered out? Do automated tests get filtered out?

### DISCUSSION THREAD

**Discussion by edgardcooper-lunadelaet** [vercel/community#5525] [Help] [answered]
### Summary

Since recently, we are using Vercel Web Analytics. As mentioned on [the documentation page](https://vercel.com/docs/analytics#bots): "web Analytics does not count traffic that comes from automated processes or accounts. This is determined by inspecting the [User Agent](https://developer.mozilla.org/docs/Web/HTTP/Headers/User-Agent) header for incoming requests."

We are wondering which User Agents get filtered out from counting as "visitors". We are especially curious if automated tests (like Checkly/Playwright) get filtered out of the results. 

I found [this thread](https://github.com/orgs/vercel/discussions/1021#discussioncomment-4038017) from 2 years ago, where they mentioned "I don't believe automated testing counts as bot traffic". Is this still the case 2 years later?

### Steps to Reproduce

/

**Comment by versecafe** [comment, marked as answer]
If your automated testing is coming from something like checkly spinning up a playwrite instance then it will appear in your analytics data, this is as of December 2023 last time I checked so I doubt it has changed in the past three weeks.

This also likely applies to GitHub CIs doing similar behaviour.

You'll likely see a large number of hits from GNU/Linux with Chrome being the browser

<img width="783" alt="image" src="https://github.com/vercel/community/assets/147033096/d924d577-851e-42e1-941d-f9347a2fc05c">
